test/parallel/test-https-simple.js

```diff
@@ -66,16 +56,15 @@ server.listen(0, function() {
   };
   noCertCheckOptions.Agent = new https.Agent(noCertCheckOptions);
 
-  const req = https.request(noCertCheckOptions, function(res) {
+  const req = https.request(noCertCheckOptions, function(res){
     let responseBody = '';
     res.on('data', function(d) {
       responseBody = responseBody + d;
     });
 
-    res.on('end', function() {
+    res.on('end', common.mustCall(() => {
```

Comment:
The enclosing `https.request()` and `server.listen()` callbacks need to have `common.mustCall()` too, or there is no guarantee that this will execute.